.circleci/config.yml

```diff
@@ -1,16 +1,15 @@
 version: 2
 jobs:
   build:
-    working_directory: /usr/src/app
     docker:
-      - image: banian/node
+      - image: circleci/node
```

Comment:
👍 

to be aligned with Nuxt, it must be better to set `circleci/node:latest` 

see https://github.com/nuxt/nuxt.js/blob/dev/.circleci/config.yml#L6